building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -1,6 +1,28 @@
 #!/bin/bash
-set -eu
+set -e -u
+rm -f resources/GIT_VERSION
```

Comment:
There's no need to remove this file explicitly; ` >resources/GIT_VERSION` will overwrite it.

Reply:
We either need to remove it or gitignore `resources/GIT_VERSION` since we are checking if the working directory is clean immediately after. Which is preferable?

Reply:
gitignoring it seems like the right solution. We wouldn't want to check it in, anyway.